objectPerformance/src/turnon_collection.py

```diff
@@ -325,23 +325,21 @@ def _skim_to_hists(self):
                 bins=self.bins
             )
 
-    def _remove_inner_nones_zeros(self, arr):
+    def _remove_inner_nones(self, arr):
         sel_arr_not_none = ~ak.is_none(arr, axis=-1)
-        sel_arr_not_zero = ak.num(arr) > 0
```

Comment:
Are you sure this is not needed? We should at least check that the object performance does not change anytime we include this kind of modification.

Reply:
I think this is related to the other change you commented on above. I will try to find some change log where we can see what the cause of this change is. The objectPerformance plots are reproduced with these changes and the new awkward version.

Reply:
I researched a bit but was not able to find anything. But evidently there was a change. What do you suggest moving forward?